src/components/OSCALControlProse.js

```diff
@@ -384,6 +385,13 @@ export function OSCALReplacedProseWithByComponentParameterValue(props) {
   // https://github.com/EasyDynamics/oscal-react-library/issues/499
   const statementByComponentDescription =
     statementByComponent?.description || null;
+  const statementByComponentDescriptionMarkup =
+    statementByComponentDescription ? (
+      <OSCALMarkupMultiLine
+        text={statementByComponentDescription}
+        paragraphComponent="p"
```

Comment:
I thinking mapping `p -> p` is a bit weird here. This goes back to my original comment suggesting the checking for `paragraphComponent` to be `null`. I don't think we should force a rewrite of the `p` tag.